src/uct/tcp/tcp_cm.c

```diff
@@ -183,8 +184,10 @@ ucs_status_t uct_tcp_cm_send_event(uct_tcp_ep_t *ep,
         conn_pkt->event      = UCT_TCP_CM_CONN_REQ;
         conn_pkt->iface_addr = iface->config.ifaddr;
         conn_pkt->conn_sn    = ep->conn_sn;
+        conn_pkt->conn_to_ep = !!(ep->flags & UCT_TCP_EP_FLAG_CONNECT_TO_EP);
         ucs_assert(ep->conn_sn < UCT_TCP_CM_CONN_SN_MAX);
     } else {
+        ucs_assert(!(ep->flags & UCT_TCP_EP_FLAG_CONNECT_TO_EP));
```

Comment:
is this really true?

Reply:
yes, other events haven't to be sent in case of CONNECT_TO_EP